astronomer/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -22,10 +22,9 @@ def __init__(
         wasb_conn_id: str = "wasb_default",
         public_read: bool = False,
     ) -> None:
-        super().__init__()
```

Comment:
Why are we removing `super().__init__()` ?

Reply:
If you look at parent class, you see that you already do everything it does in this __init__